README.md

```diff
@@ -957,6 +950,13 @@ function makeStore(storeClass: Class<Store>): Store {
 
 [⇧ back to top](#flows-utility-types)
 
+### mixed
+
+An arbitrary type that could be anything (same as `unknown`)
```

Comment:
still missing 2 spaces on the end of this line

Reply:
What spaces?

Reply:
@goodmind If you don't know how Markdown formatting works, if you add 2 spaces at the end, then it will add a line break. It's the same as adding <br />. I need that because url is stacked on the same line as description which is wrong